Integrations/src/main/java/io/deephaven/integrations/python/PythonDeephavenSession.java

```diff
@@ -56,9 +56,7 @@ public class PythonDeephavenSession extends AbstractScriptSession<PythonSnapshot
     private static final Logger log = LoggerFactory.getLogger(PythonDeephavenSession.class);
 
     private static final String DEFAULT_SCRIPT_PATH = Configuration.getInstance()
-            .getProperty("PythonDeephavenSession.defaultScriptPath")
-            .replace("<devroot>", Configuration.getInstance().getDevRootPath())
-            .replace("<workspace>", Configuration.getInstance().getWorkspacePath());
+            .getStringWithDefault("PythonDeephavenSession.defaultScriptPath", ".");
```

Comment:
Are we confident about making `.` the new default for all the script sessions? What testing have you done?

Reply:
This previous defaults were `<X>.defaultScriptPath=<devroot>`; and by default, `devroot=.`. So in theory, it's the same defaults as before, just a layer of indirection removed.

I think the defaultScriptPath logic could use serious reworking - I'm not sure it makes sense anymore if we have a better way w/ application mode. There's a comment to this effect in `io.deephaven.engine.util.ScriptFinder#findScriptEx`

I've run locally natively in python and groovy. I'll do testing in docker as well. I'll start a nightly as well.

Reply:
Everything I've run seems to work. nightly is passing except I hit this https://github.com/deephaven/deephaven-core/issues/2732

Reply:
Also run python embedded server, work fine.